docs/api/public_php_api.md

```diff
@@ -702,6 +702,10 @@ Query object contains two properties you can set criteria on, `filter` and `quer
 
 #### Performing a simple full text search
 
+!!! tip "Checking feature support per search engine"
+
+    To find out if a given search engine supports advanced full text capabilities, use the [`supports($capabilityFlag)`](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/SearchService.php#L187-L197) method.
```

Comment:
example could be expanded to `$searchService->supports($capabilityFlag)`

Reply:
Side: @lserwatka you and team should probably decide on a API doc feature, links below point to Sami, others to doxygen.